include/libfluid-base/OFClient.hh

```diff
@@ -1,3 +1,18 @@
+// Copyright (c) 2014 Open Networking Foundation
+// Copyright 2021 The Alcor Authors
```

Comment:
We could use "Copyright 2020 Futurewei Cloud" throughout (previously we did use "Alcor Authors" but changed earlier this year).

Reply:
Renamed